base/intfuncs.jl

```diff
@@ -35,6 +48,19 @@ function gcd{T<:Union{Int64,UInt64,Int128,UInt128}}(a::T, b::T)
 end
 
 # explicit a==0 test is to handle case of lcm(0,0) correctly
```

Comment:
`lcm` - maybe rearrange the function definition to use the longer multiline form so this comment about the "explicit a==0 test" is closer to the corresponding code?